seqr/views/utils/dataset_utils.py

```diff
@@ -34,44 +34,53 @@ def load_mapping_file_content(file_content):
     return id_mapping
 
 
```

Comment:
this file got a bit refactored because previously you could add samples for search in one project at a time and samples for RNA in multiple projects at once. Since for this new command we wanted to be able to add multiple projects at once, I just combined as much of the logic as possible between the search and RNA samples